packages/cli/.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "rules": {}
+    },
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {}
+    },
+    {
+      "files": ["*.js", "*.jsx"],
+      "rules": {}
+    },
+    {
+      "files": ["*.json"],
+      "parser": "jsonc-eslint-parser",
+      "rules": {
+        "@nx/dependency-checks": [
+          "error",
+          {
+            "ignoredDependencies": [
+              "vite",
+              "@nx/vite",
+              "@quality-metrics/eslint-plugin",
+              "@quality-metrics/lighthouse-plugin"
```

Comment:
Not sure what the purpose of this is. Don't you usually ignore all deps for linting?

Reply:
The rule checks that every imported package is listed in the `package.json`, which is pretty cool. E.g. I would get a lint error if I don't have `bundle-require` in the `cli`'s `package.json`.

However, I had to add some exceptions to make the linter pass:
- Adding the `@quality-metrics/eslint-plugin` and `@quality-metrics/lighthouse-plugin` is temporary and won't be needed once we move out the test files which import them in #19.
- For some reason, the Nx rule gets tripped up by the imports in `vite.config.ts`. There's [a related issue for it in Nx](https://github.com/nrwl/nx/issues/17905) and even [a merged PR](https://github.com/nrwl/nx/pull/17930), but it doesn't seem to be in the latest stable version yet.

Reply:
Super cool